CMakeLists.txt

```diff
@@ -833,6 +834,10 @@ IF(WIN32)
      ELSE()
           SET(ISS_SPOUT "; Spout support is disabled")
      ENDIF(ENABLE_SPOUT)
+     SET(SHOWMYSKY_LIBRARY "${ShowMySky_INCLUDE_DIRECTORIES}/../ShowMySky.dll")
```

Comment:
Hmm... Shouldn't be this line inside `IF(ENABLE_SHOWMYSKY)/ENDIF()` box?

Reply:
I had the same thought. If the variable is never used, it would not harm much though. 

Finally, deployment (building an installer including the ShowMySky.dll) works on private builds in QtCreator!

Reply:
Yeah, this doesn't make a difference. Initially I wanted to check that the file exists, that's why I actually created this variable. But then I thought that ISS will complain anyway, so this check would be superfluous. Now the variable is not useful outside the *if*, I can move it in.